app.py

```diff
@@ -190,7 +190,7 @@ def put(self, meme_id):
         return send_success("Approved meme %s" % meme_id)
 
 
-class MemeVotingResource(Resource):
+class MemeUnvotingResource(Resource):
```

Comment:
If we do the above approach of sending the vote type in the request body, we can re-combine the "Unvoting" resource with the "Voting" resource

Reply:
API endpoints have been combined into MemeVotingResource.